Update fastbuild readme (#1522)
spatialos/UnrealGDK: SpatialGDK/Extras/fastbuild/README.md

# Introduction
[FASTBuild](http://www.fastbuild.org/docs/home.html) is a distributed build and caching system.

The GDK for Unreal integrates with the Unreal Build Tool, using [Unreal_FASTBuild](https://github.com/liamkf/Unreal_FASTBuild)
as the foundation.

# Cache location 

To use Fastbuild outside Improbable, you need to configure your own caching location, where compilation results can be shared across users.

Follow [these instructions](http://www.fastbuild.org/docs/features/caching.html) to set up your cache and update the `$fileshare` variable in `install.ps1` with your value.

# Cache location (internal to Improbable)

> Improbable's fileshare is `\\lonv-file-01` which is set as the default value in the Installation script. To use it, you first need to authorise access to this network drive:

Reviewer: ```suggestion
Improbable's cache location is `\\lonv-file-01` which is set as the default value in the installation script. To use it, you first need to authorise access to this network drive:
```

1. Enter `\\lonv-file-01` your Windows Explorer address bar

> 1. Authenticate with your Windows username and password (not the PIN)

Reviewer 2: ```suggestion
1. Authenticate with your Windows username and password (not the PIN). Make sure to set `Remember my credentials` so this works after a restart.
```

> 

Reviewer 3: ```suggestion
The install script is located at `\\lonv-file-01\Fastbuild\install.ps1`.
```

Author: the install scripts are also in the GDK repo (same folder as the readme) which should be referenced here, so that external users can use them too
